File: MinimalChipViewModel.java
package org.thoughtcrime.securesms.trustedIntroductions;

import androidx.lifecycle.ViewModel;

import java.util.ArrayList;
import java.util.List;
import java.util.Objects;

public class MinimalChipViewModel extends ViewModel {
  private final List<String> selected;

  MinimalChipViewModel(){
    selected = new ArrayList<>();
  }

  void add(String contact){
    selected.add(contact);
  }

  void remove(String contact){
    Object toRemove = null;
    for(String s: selected){
      if (contact.equals(s)){
        toRemove = s;
      }
    }
    if (toRemove != null){
      selected.remove(toRemove);
    }
  }

}

File: MinimalChipViewModel.kt
package org.thoughtcrime.securesms.trustedIntroductions

import androidx.lifecycle.ViewModel


class MinimalChipViewModel : ViewModel() {

  private var selected: ArrayList<String> = ArrayList<String>()

  fun add(selectedString: String){
    selected.add(selectedString)
  }

  fun remove(selectedString: String){
    selected = selected.filterNot { str -> str.equals(selectedString) } as ArrayList<String>
  }

  fun numberOfChips(): Int{
    return selected.size
  }
}